sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/SchemaUtil.java

```diff
@@ -313,11 +311,11 @@ private static <InputT, BaseT> ResultSetFieldExtractor createLogicalTypeExtracto
   /** Convert SQL date type to Beam DateTime. */
   private static ResultSetFieldExtractor createDateExtractor() {
     return (rs, i) -> {
-      Date date = rs.getDate(i, Calendar.getInstance(TimeZone.getTimeZone(ZoneOffset.UTC)));
+      java.time.LocalDate date = rs.getObject(i, java.time.LocalDate.class);
```

Comment:
can we change this to an import?

Reply:
Unfortunately there is import of `org.joda.time.LocalDate` and used everywhere. There is an issue #19215 to eliminate joda.time in release 3.0.0

Reply:
Gross, but understandable. Can you add a TODO here linked to that issue to clean this up?